core/state/snapshot/wipe_test.go

```diff
@@ -30,8 +30,9 @@ import (
 	"math/rand"
 	"testing"
 
-	"github.com/ava-labs/coreth/core/rawdb"
+	customrawdb "github.com/ava-labs/coreth/plugin/evm/rawdb"
```

Comment:
This doesn't need to be renamed. As for whether it should still be, I don't think so as in this context it doesn't matter if the reader knows the exact package of the implementations.

[Relevant entry in Google style guide](https://google.github.io/styleguide/go/decisions#:~:text=Imports%20should%20only%20be%20renamed%20to%20avoid%20a%20name%20collision%20with%20other%20imports.)

Reply:
It does I think, given it would conflict with the libevm import:

```go
	customrawdb "github.com/ava-labs/coreth/plugin/evm/rawdb"
	"github.com/ava-labs/libevm/common"
	"github.com/ava-labs/libevm/core/rawdb"
```